src/aosm/README.md

```diff
@@ -67,7 +67,10 @@ image that would be used for the VNF Virtual Machine.
 
 #### CNFs
```

Comment:
This readme could do with a bit more of a description of what the user needs to do in advance of using the CLI.  For example have the helm chart on disk and the image in an ACR??